glue/__init__.py

```diff
@@ -118,6 +118,11 @@ def make_selector_func(roi):
     from .qt.custom_viewer import CustomViewer
     return CustomViewer.create_new_subclass(name, **kwargs)
 
+# Load built-in plugins
+from .config import load_plugins
+load_plugins()
```

Comment:
@ChrisBeaumont - ah right, indeed it would be nice to avoid the Qt import. I will think about this a bit more.